modules/videoio/src/cap_mfx_common.hpp

```diff
@@ -12,12 +12,22 @@
 #include <fstream>
 #include <sstream>
 
-#include <mfxcommon.h>
-#include <mfxstructures.h>
-#include <mfxvideo++.h>
-#include <mfxvp8.h>
-#include <mfxjpeg.h>
-#include <mfxplugin++.h>
+#ifdef HAVE_ONEVPL
```

Comment:
Could you please use a single header file: mfx.h which includes all oneVPL header files?